app/views/two_factor_authentication/webauthn_verification/show.html.erb

```diff
@@ -17,7 +17,7 @@
   <%= hidden_field_tag :signature, '', id: 'signature' %>
   <%= hidden_field_tag :client_data_json, '', id: 'client_data_json' %>
 
-  <div id='webauthn-auth-in-progress'>
+  <div id='webauthn-auth-in-progress' data-webauthn-not-enabled-url='<%= login_two_factor_options_path %>'>
```

Comment:
We've been using `content_tag` to handle some of the escaping, etc etc, so we could rewrite this as:

```suggestion
  <%= content_tag :div, id: 'webauthn-auth-in-progress', data: { webauth_not_enabled_url: login_two_factor_options_path } do %>
```

(also have to add an `end` block at the end too)